### PR TITLE
Add remove_from_all method to `Corpus` trait

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

**Comment by tokatoka** [comment]
@R9295 Can you check this?

**Comment by R9295** [comment]
Looks like the LibfuzzerCorpus & ArtifactCorpus also need remove_from_all implementations.

LGTM otherwise

File: libafl/src/corpus/cached.rs [hunk 1]
```diff
@@ -115,6 +115,13 @@ where
         Ok(testcase)
     }
 
+    /// Removes an entry from the corpus, returning it if it was present; considers both enabled and disabled testcases.
+    fn remove_from_all(&mut self, idx: CorpusId) -> Result<Testcase<Self::Input>, Error> {
```
Comment (domenukk) [inline libafl/src/corpus/cached.rs:119]: Shouldn't `remove` just remove from wherever? Why does it make a difference if the testcase is disabled or not?

Comment (tokatoka) [reply libafl/src/corpus/cached.rs:119]: every thing without `all` does not touch disabled entry

Comment (domenukk) [reply libafl/src/corpus/cached.rs:119]: IMHO remove should remove, no matter what

Comment (tokatoka) [reply libafl/src/corpus/cached.rs:119]: Ok i'll change

**Pull request merged**